dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtilTest.java

```diff
@@ -792,8 +792,12 @@ public void testWriteGeoLocationElement2() throws XMLStreamException, IOExceptio
         writer.close();
         Assertions.assertEquals("<geoLocations>"
                         + "<geoLocation>"
-                        + "<geoLocationBox>34 45 89 23</geoLocationBox>"
-                        + "</geoLocation></geoLocations>",
+                        + "<geoLocationBox>34 23 89 45</geoLocationBox>"
+                        + "</geoLocation>"
+                        + "<geoLocation>"
+                        + "<geoLocationPoint>45.21 23.1</geoLocationPoint>"
+                        + "</geoLocation>"
+                        + "</geoLocations>",
```

Comment:
Maybe we could also add case where we have polygon with more than 5 points in `geographicCoordinates` (something more complex that rectangle)?